doc/nrf/app_dev/device_guides/nrf70/fw_patches_ext_flash.rst

```diff
@@ -91,24 +91,7 @@ The following configuration options are available:
 * :kconfig:option:`CONFIG_NRF_WIFI_FW_FLASH_CHUNK_SIZE` - Defines the size of the chunks used to read the firmware patches from the external non-XIP memory.
   The default value is 8192 bytes.
 
-The external memory partition name must be defined in the devicetree or in the partition manager configuration file.
-
-* ``nrf70_fw_partition`` - Defines the name of the external memory partition that stores the firmware patches.
-  This must be defined in the devicetree, for example:
-
-.. code-block:: dts
-
-    &flash0 {
-        partitions {
-            compatible = "fixed-partitions";
-            #address-cells = <1>;
-            #size-cells = <1>;
-            nrf70_fw_partition: partition@0 {
-                label = "nrf70_fw_partition";
-                reg = <0x00000000 DT_SIZE_K(128)>;
-            };
-        };
-    };
+The external memory partition name must be defined in the partition manager configuration file as follows:
```

Comment:
Is it duplicated in line 97 which is mentioning the same thing?

Reply:
not quite, 97 says that the partition must be defined in the partition manager file, I will move the note before that.